include/erlavro.hrl

```diff
@@ -48,7 +48,9 @@
          N =:= ?AVRO_DOUBLE  orelse
          N =:= ?AVRO_BYTES)).
 
+-ifndef(AVRO_REQUIRED).
 -define(AVRO_REQUIRED, erlang:error({required_field_missed, ?MODULE, ?LINE})).
+-endif.
```

Comment:
Thanks, but based on recent discussion in the issue thread, please drop the `-ifndef` wrapper and just delete `?MODULE,` from the error term. Please also add a changelog entry for tag 2.9.4 with a brief note about improving compatibility with Elixir or something like that, and squash to a single commit.